### PR TITLE
Feature/gulp server

### PR DESCRIPTION
**Pull request by xaun**
@ouranos please review :balloon: 

@cesar-tonnoir :beer:

### CODE REVIEW

**Comment by ouranos** [comment]
Looks good.  Maybe add some doc in the readme?

If you want a more granular watch for later you can have a look at the mnoe Next project.

**Comment by xaun** [comment]
Docs added, @cesar-tonnoir this would be good to merge.. live reload :tada:

File: Changelog.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # master | Ongoing development
 
 ### Adds
+> Gulp Server with live reload for Developer Workspace!
```
Comment (cesar-tonnoir) [inline Changelog.md]: @xaun I fixed the changelog replacing these ">" with "-". please update

**Comment by cesar-tonnoir** [comment]
:beers: thanks @xaun and @leomao10 for your contribution!

**Pull request merged**